pages/blog/openapi-vs-asyncapi-burning-questions.md

```diff
@@ -0,0 +1,122 @@
+---
+type: Engineering
+cover: /img/posts/openapi-vs-asyncapi-burning-questions/asyncapi-openapi-post_pic-01.webp
```

Comment:
It doesn't look too well when the blog post images are used as a cover photo 👇🏼 

![Screenshot 2021-08-04 at 07 30 37](https://user-images.githubusercontent.com/6995927/128127184-e836e262-4938-434e-a0c9-84fd40c74bca.png)
![Screenshot 2021-08-04 at 07 30 21](https://user-images.githubusercontent.com/6995927/128127191-a5206259-55ac-4d9f-832a-cd8c2585c918.png)

would be good to pick some dedicated cover. Maybe do a spoiler and provide as cover `asyncapi-openapi-post_pic-14.webp` but with icons not next to each other but one under another?

Reply:
Thanks, changed image.